### ISSUE TITLE
onNotification does not trigge only in ios when app running background mode

### ISSUE THREAD

**Issue by vanhungoz** [zo0r/react-native-push-notification#746]
react-native : 0.53.3
react-native-push-notification: 3.0.2 
 in my app, when app running in background mode. I have a remote notification, user tap it then my app was opened. but onNotification function was not trigged.

**Comment by batuhanorhn** [comment]
+1

**Comment by tatsna** [comment]
+1

**Comment by batuhanorhn** [comment]
PushNotification.configure in add;

onRegister: function (token) {
         console.log('TOKEN:', token);

         PushNotification.popInitialNotification(notification => {
              console.log(notification); 
          })
       },

**Comment by Gp2mv3** [comment]
I think it's a dupplicate of #547. Feel free to reopen this if not.

**Comment by iamrutvik** [comment]
React Native Push Notifications clearly states that we need to call configure outside of the React Native Lifecycle. This is very important. 

https://product.farewell.io/visible-react-native-push-notifications-that-work-on-both-ios-and-android-5e90badb4a0f

https://prnt.sc/k61hz7

**Comment by Scpchao** [comment]
@iamrutvik Brother, this way you can't trigger onNotification, see you reply a lot of posts, this is the answer, how do you do it?

**Comment by nairabhijit** [comment]
@Gp2mv3 The thread reference you posted discuss about issues on Android, I am facing this issue on iOS. In my case when the application is in a killed state, I receive notifications but when I tap on it the onNotification handler isn't called. Also I have kept the PushNotification.configure outside of the react native lifecycle, I tried everything recommended in several Github posts but without success. Any help would be highly appreciated.
